src/test/test-cpu-set-util.c

```diff
@@ -4,6 +4,10 @@
 #include "cpu-set-util.h"
 #include "tests.h"
 
+#ifndef __GLIBC__
+typedef unsigned long __cpu_mask;
+#endif
```

Comment:
hmm, this might not be correct. I think we should just skip the relevant test on non-glibc, that would be safer. 

it's better to not test this facet than to test the wrong thing